python/cudf/cudf_pandas_tests/test_cudf_pandas_fallback.py

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from cudf.pandas import LOADED
+
+if not LOADED:
+    raise ImportError("These tests must be run with cudf.pandas loaded")
+
+import numpy as np
+import pandas as pd
+
+
+@pytest.fixture(autouse=True)
+def fail_on_fallback(monkeypatch):
+    monkeypatch.setenv("CUDF_PANDAS_FAIL_ON_FALLBACK", "True")
```

Comment:
Could this just be set in our existing collection of tests in `python/cudf/cudf_pandas_tests/test_cudf_pandas.py`?

Reply:
Yes it could, but I thought it would be better to keep the tests apart because they're testing different things. In `test_cudf_pandas.py`, we're testing whether results between `cudf.pandas` and `pandas` are equal. In `test_cudf_pandas_fallback.py`, we're testing for no fallback.

Reply:
OK sounds good to keep these tests separate then. Could we rename this file to include `no_fallback.py` to make that clear?